packages/client/src/features/accessibility/view-key-tools/movement-key-tool.ts

```diff
@@ -68,6 +73,16 @@ export class MoveKeyListener extends KeyListener {
         }
     }
 
+    registerShortcutKey(): void {
+        this.tool.actionDispatcher.onceModelInitialized().then(() => {
+            this.tool.actionDispatcher.dispatchAll([
+                SetKeyShortcutAction.create(this.token, [
+                    { shortcuts: ['⬅ ⬆ ➡ ⬇'], description: 'Move element or viewport', group: 'Move', position: 0 }
```

Comment:
I'd suggest to add spaces in between the arrows
```suggestion
                    { shortcuts: ['⬅  ⬆  ➡  ⬇'], description: 'Move element or viewport', group: 'Move', position: 0 }
```